Portfolio/index.css

```diff
@@ -111,32 +116,174 @@ a {
     text-align: center;
 }
 
-.about{
+.about {
     display: flex;
     flex-direction: column;
     text-align: center;
     justify-content: center;
     padding: 2rem;
 }
-.about-title{
+
+.about-title {
     font-size: 2.5rem;
     letter-spacing: 0.03em;
     margin: 2rem;
 }
-.about-article{
+
+.about-article {
     margin: 2rem;
-    
+
     font-size: 1.9rem;
     letter-spacing: 0.035em;
     text-align: left;
-    
+    padding: 0px 8rem;
+
 }
-article{
-    font-family: "Roboto", sans-serif; 
+
+article {
+    font-family: "Roboto", sans-serif;
     color: #9ABEE9;
 
 }
-span{
-    font-family: "Roboto", sans-serif; 

```

Comment:
Consider uncommenting the height property for .contact-form to ensure all content is visible without overflow